install/python_requirements.txt

```diff
@@ -3,6 +3,7 @@
 
 #Appium-Python-Client 
 Sphinx
```

Comment:
0.6.1.0 version should be without sphinx. 
Holger's new toolchain doesn't need this. But I don't know if all repos are are changed to the new toolchain.
So better we let it for 0.6.0.n as it is.